pkg/operator/k8sutil/pod_test.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package k8sutil_test
```

Comment:
just one more comment on a rook preferred pattern... rather than declaring the test project `k8sutil_test`, if you declare the test in the `k8sutil` package,  you won't need the k8sutil prefix anywhere and the new variables such as `RepoPrefixEnvVar` can be declared as private package variables (`repoPrefixEnvVar`).